pkg/network/setup/configstate_test.go

```diff
@@ -173,8 +176,8 @@ var _ = Describe("config state", func() {
 	When("with multiple interfaces", func() {
 		BeforeEach(func() {
 			nics = append(nics,
-				podNIC{podInterfaceName: testNet1},
-				podNIC{podInterfaceName: testNet2},
+				podNIC{podInterfaceName: testNet1, vmiSpecNetwork: &v1.Network{Name: testNet1}},
```

Comment:
I think `podInterfaceName` can be removed here.

Reply:
Right, done.